### ISSUE TITLE
Version info fails if Bible hasn't been downloaded to device

### ISSUE THREAD

**Issue by mondele** [unfoldingWord-dev/uw-android#183]
On Android only (confirmed this is not an issue on iOS) Version Information fails when the Bible in question hasn't been downloaded to the device. If the Bible has been downloaded, it succeeds.

https://files.slack.com/files-tmb/T04PNHLBY-F9HA11RUM-1931b0863c/screenshot_20180301-150407_1024.png

https://files.slack.com/files-tmb/T04PNHLBY-F9HA17WP7-e85587d007/screenshot_20180301-150614_1024.png

**Comment by jag3773** [comment]
@mondele you mean the content signature verification fails?

**Comment by mondele** [comment]
I'm not sure what to call it. It's the error shown in the Info window in the screenshots above. When the book is downloaded, it reads "All Content is Verified".

**Comment by jag3773** [comment]
Right, it's trying to provide a cryptographic verification of the content, but the content does not yet exist on the device, so it fails.
